root/tree/addresses/ConfigRecord.h

```diff
@@ -9,7 +9,7 @@ class ConfigRecord : public RecDataRecord<RecHeader>
 public:
   ConfigRecord() {}
   ConfigRecord(const RecHeader& header) : RecDataRecord<RecHeader>(header){} 
-  virtual ~ConfigRecord() {}
+  ~ConfigRecord() override {}
```

Comment:
I think you also need to change the `ClassDef` to `ClassDefOverride`

Reply:
Thanks! I have updated the PR now. Took me some iterations because some classes needed to keep `ClassDef` because they didn't inherit from TObject.